alert-platform/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java-platform'
+}
+
+dependencies {
+    constraints {
+        api 'com.github.springtestdbunit:spring-test-dbunit:1.3.0'
+        api 'org.testcontainers:postgresql:1.12.5'
+        api 'org.apache.activemq:activemq-broker:5.15.10'
+
+        api 'org.jsoup:jsoup:1.9.2'
+        api 'com.jayway.jsonpath:json-path:2.3.0'
+        api 'com.sun.mail:javax.mail:1.6.2'
+        api 'javax.mail:javax.mail-api:1.6.2'
+
+        api 'org.postgresql:postgresql:42.2.8'
+        api 'org.liquibase:liquibase-core:3.8.9'
```

Comment:
These should be `runtime` dependencies.

Reply:
In platform, everything is api - they should be runtime where they are included.

Reply:
Nope, sorry, you're right. I'll check.